embed_grader/serapis/models.py

```diff
@@ -283,22 +283,12 @@ def can_access_grading_details_by_user(self, user):
     def can_access_grading_script_by_user(self, user):
         return user.has_perm('create_assignment', self.assignment_fk.course_fk)
 
-    def retrieve_assignment_task_files(self, user):
-        viewing_scope = self.assignment_fk.viewing_scope_by_user(user)
-        assign_task_file_list = AssignmentTaskFile.objects.filter(assignment_task_fk=self.id)
+    def retrieve_assignment_task_files_url(self, user):
```

Comment:
Great! Just make sure that you use `find` to replace all the call-sites

Reply:
Yup, I did